src/lib/gateway/Gateway.ts

```diff
@@ -0,0 +1,98 @@
+import { RequestHandler, IdKeyed } from '@klasa/request-handler';
+import Collection from '@discordjs/collection';
+import { GatewayStorage } from './GatewayStorage';
+import { Settings } from '../settings/Settings';
+import { Client } from '../types';
+
+export class Gateway extends GatewayStorage {
+
+	/* eslint-disable no-invalid-this */
+	/**
+	 * The cached entries for this Gateway or the external datastore to get the settings from.
+	 */
```

Comment:
```ts
/** The cached entries for this Gateway or the external datastore to get the settings from. */
```
One liners are better. Apply to all TSDoc where it's just a description.

Reply:
![image](https://user-images.githubusercontent.com/24852502/70475561-10c62a00-1ad5-11ea-8895-40282c5cc84f.png)